metadata/pkg/service/service.go

```diff
@@ -157,7 +158,18 @@ func (f *metadataService) SyncMetadata(ctx context.Context, in *pb.SyncMetadataR
 }
 
 func (f *metadataService) ListMetadata(ctx context.Context, in *pb.ListMetadataRequest, out *pb.ListMetadataResponse) error {
-	log.Info("Received GetMetadata request in metadata service.")
+	log.Info("received GetMetadata request in metadata service.")
```

Comment:
plz be cosistent with either starting char should be caps or small.

Reply:
keeping all info logs in lower case 
keeping all error logs starting with uppercase